src/rewrite/vct/rewrite/SimplifyNestedQuantifiers.scala

```diff
@@ -228,6 +228,18 @@ case class SimplifyNestedQuantifiers[Pre <: Generation]()
     )(contract.blame)(contract.o)
   }
 
+  private def hasTriggers(e: Binder[Pre]): Boolean =
+    e match {
+      case Forall(_, triggers, body) =>
```

Comment:
Honestly maybe we could argue for a "Quantifier" trait that extends binder to generalize forall/exists... Not sure if there would be any use elsewhere in vercors tho